### ISSUE TITLE
Unable to change crop area on Android

### ISSUE THREAD

**Issue by johnsmathew** [CrackerakiUA/ui-cropper#11]
I'm building an android app using cordova and angularjs. Everything is working fine when I test the app on my browser. But when I build the APK and test on android emulator or on android device, I can't change the crop area. I am able to select an image and the image successfully loads in canvas. The preview is also updated. But when I drag on canvas to change the crop area, the page scrolls without causing any change to the canvas. 

I did a console.log on `onMouseMove`, `onMouseDown` and `onMouseUp` methods in ui-cropper.js. The events are getting triggered.

Any idea what would be wrong?

**Comment by CrackerakiUA** [comment]
I am building also an app on android cordova, but at the moment it's on pause, when i have more time i will test it and will reply to you. If you keep digging, try to use $timeout on the place of apply those events, maybe angular don't trigger clock there.

**Comment by johnsmathew** [comment]
What confuses me is that it works fine when I run cordova app on my desktop browser. The issue occurs only when I run the apk on android. I just tried the sample codepen for ui-cropper on the same android device in browser. And it works!

I'll try building another app with just one page to test the cropping. Maybe it is some compatibility issues with other libraries I use.

**Comment by johnsmathew** [comment]
@CrackerakiUA  It is some incompatibilities with other libraries I use. I made a new cordova app with just ui-cropper and it is working fine.

**Comment by johnsmathew** [comment]
@CrackerakiUA I found the real cause. I had set a zoom on the body tag to fit the page to the device depending on `window.devicePixelRatio`. Seems like the coordinates from the touch event were manipulated by this zoom value. When running on my desktop browser, `window.devicePixelRatio` is 1, so there was no issues. For now, I divided `pageX` and `pageY` with the zoom value and crop is working. I have opened a new issue for this - #13.
